nautilus_trader/core/nautilus_pyo3.pyi

```diff
@@ -1973,6 +1985,8 @@ class OrderInitialized:
     @classmethod
     def from_dict(cls, values: dict[str, str]) -> OrderInitialized: ...
     def to_dict(self) -> dict[str, str]: ...
+    @property
+    def order_type(self) -> str: ...
```

Comment:
What was the motivation to add this? it's not always easily available at event generation.

Reply:
To separate them at runtime, so that we know how to properly transform them